scalyr_agent/agent_main.py

```diff
@@ -884,6 +888,8 @@ def __run(self, controller):
         last_overall_stats_report_time = time.time()
         # We only emit the bandwidth stats once every minute.  Track when we last reported it.
         last_bw_stats_report_time = last_overall_stats_report_time
+        # We only emit the copying_manager stats once every 5 minutes.  Track when we last reported it.
+        last_copy_manager_stats_report_time = last_overall_stats_report_time
```

Comment:
To improve readability here, let's declare a variable called `current_time` and set `time.time()` and use it in all of the above statements.  It looks a little odd when reading this line without that context.

Reply:
We have a `self.__start_time` set a bit above this, I think that would make sense to use here. Naming wise it seems pretty clear.